File: fibonacci_sequence.c
#include<stdio.h>

int main()
{
    int terms;
    printf("Enter the number of terms of Fibonacci Sequence you want to show: ");
    scanf("%i",&terms);

    int f1=0,f2=1;
    printf("0\n");
    printf("1\n");

    for (int i=1; i<=terms-2; i++)
    {
        int sum=f1+f2;
        printf("%i\n",sum);
        f1=f2;
        f2=sum;
    }

    return 0;
}

File: fibonacci_sequence.py
f1=0
f2=1

number=int(input("Enter the number of terms of the Fibonacci Series you want to show: "))

print(f1)
print(f2)

for i in range (0,number-2):
	fsum=f1+f2
	print(fsum)
	f1=f2
	f2=fsum
	i+=1
